images/macos/provision/core/python.sh

```diff
@@ -16,6 +13,13 @@ pip install --upgrade pip
 echo "Install Python2 certificates"
 bash -c "/Applications/Python\ 2.7/Install\ Certificates.command"
 
+# Remove /usr/local/bin/2to3 symlink created by Python2 since it conflicts with symlink from Python3
+# https://github.com/actions/virtual-environments/issues/2322
+rm -rf /usr/local/bin/2to3
```

Comment:
We shouldn't use this solution. It is unreliable and will bring more problems and maintenance in future.
What if brew updates "python@3.9" formula and add one more symlink? You will have to remove it too and it will break customers again.

Reply:
I think the correct solution will be:
- Install Python2 from package
- Install `python@3.9` from brew (with ignoring errors)
- run `brew link python@3.9`

This approach will override `2to3` symlink and any other symlinks that can be added in future.

Reply:
Something like `brew install <formula> || brew link --overwrite <formula>`

Reply:
@maxim-lobanov agree, that makes sense